### ISSUE TITLE
Can not connect to CH340G usb to serial with octopi image 

### ISSUE THREAD

**Issue by Custom3DProducts** [guysoft/OctoPi#496]
The standard octopi image does not include the drivers needed to connect to the CH340G usb to serial chip. 
This makes some boards and ramps configurations unusable.
know printer board with CH340G is MKS GEN L

**Comment by guysoft** [comment]
strange,  I have a printer with MKS gen L and it works fine with Marlin.
What firmware? What OctoPi version?

**Comment by trajesus** [comment]
My anet a8 board uses the CH340G chip and my octopi can connect perfectly. However, not in the auto mode, I had to specify the baud rate and the serial port after that it's been running perfectly for almost a year :)

**Comment by guysoft** [comment]
Auto mode never worked for me, I think its dead,  @foosel?

**Comment by foosel** [comment]
No, it's not "dead". Port detection works, but it depends on there either being only one port or the printer controller to allow entering programming mode. And baudrate detection depends on the printer actually replying to probes in a timely fashion.

**Comment by guysoft** [comment]
@trajesus What is the output of lsusb?

**Comment by trajesus** [comment]
I don't have any problem with my octopi, but if you want to know the output when I arrive home I'll post it here :)

**Comment by guysoft** [comment]
Ah, then nevermind
